ksqldb-engine/src/test/java/io/confluent/ksql/function/UdfLoaderTest.java

```diff
@@ -635,7 +636,12 @@ public void shouldUseConfigForExtDir() {
         .build();
     final KsqlConfig config
         = new KsqlConfig(configMap);
-    UserFunctionLoader.newInstance(config, functionRegistry, "").load();
+    UserFunctionLoader.newInstance(
+        config,
+        functionRegistry,
+        "",
+        new Metrics()
```

Comment:
I'm just liberally creating new registries in the tests where it doesn't really matter what they are. We might be able to share some instances, but I'm in the mood to keep it simple at this point in the PR.